src/makeCalculator.js

```diff
@@ -37,7 +37,47 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  const calculator = {
+    result: 0,
+
+    operate(action, number) {
+      action(number);
+
+      return this;
+    },
+
+    reset() {
+      this.result = 0;
+
+      return this;
+    },
+
+    add(number) {
+      calculator.result += number;
```

Comment:
```suggestion
      this.result += number;
```
use `this`, as Alisa suggested: https://github.com/mate-academy/js_advanced_calculator/pull/1172#discussion_r1056800542

Reply:
Я шарю в тому. Просто чомусь, test не проходить з this. А тільки з назвою об'кту...